src/common/marketplaces/categories.js

```diff
@@ -62,6 +62,16 @@ export default [
 		svgIcon:
 			'https://dl.airtable.com/.attachments/f3f557d642883b0fb004c49dfe3b89aa/6547f314/icon-marketplace-loans.svg'
 	},
+	{
+		id: 'keyfi_individual',
+		name: 'keyfi',
+		title: 'KeyFi.ai Credentials',
+		description:
+			'Verify your Credentials to access and earn using KeyFi.ai. A First of its kind DeFi aggregator platform backed by SelfKey Credentials that allows you to liquidity mine across top DeFi platforms and earn rewards.',
+		entityType: 'individual',
+		active: featureIsEnabled('keyfi'),
+		svgIcon: 'https://dl.airtable.com/cQcRdsGeQSeM3ekJFo3E_icon-exchange-notaries.svg.png'
```

Comment:
This is bad, thous icons should be inside the source code.

Let's open an issue for it

Reply:
Agreed. Issue: https://github.com/SelfKeyFoundation/Identity-Wallet/issues/2440